storybook/.storybook/main.ts

```diff
@@ -79,19 +66,15 @@ const config: StorybookConfig = {
 
     return {};
   },
-  addons: [
-    getAbsolutePath("@storybook/addon-essentials"),
-    getAbsolutePath("@whitespace/storybook-addon-html"),
-    getAbsolutePath("@storybook/addon-a11y"),
-  ],
+  addons: ["@storybook/addon-essentials", "@whitespace/storybook-addon-html", "@storybook/addon-a11y"],
   stories: testStoryStoryV7
     ? ["../src/components/**/*.{core-,css-}stories.ts"]
     : ["../src/components/**/*.{core-,css-,}stories.ts", "../src/example-pages/**/*.ts"],
   previewHead: (head) => `
     ${head}
-    <link rel="stylesheet" href="dso-toolkit/dist/dso.css">
-    <script type="module" src="core/dso-toolkit.esm.js"></script>
-    <script src="iframe-resizer/child/index.umd.js"></script>
+    <link rel="preload" href="/static/packages/dso-toolkit/assets/fonts/Asap/Asap-Italic-VariableFont_wdth,wght.ttf" as="font" type="font/ttf" crossorigin>
+    <link rel="preload" href="/static/packages/dso-toolkit/assets/fonts/Asap/Asap-VariableFont_wdth,wght.ttf" as="font" type="font/ttf" crossorigin>
+    <link rel="preload" href="/static/packages/dso-toolkit/dist/di.svg" as="image" type="image/svg+xml">
```

Comment:
In Core Storybook:
- `dso.scss` is verplaatst naar `preview.ts`
- dso-toolkit dist (lazy loading distributie) is vervangen met `defineCustomElements()` in `preview.ts`
- `di.svg` wordt geinlined.
- iframe-resize/child is verplaatst naar `preview.ts`